### DISCUSSION TITLE
[Feature] Group places by countries

### DISCUSSION THREAD

**Discussion by lks-nbg** [immich-app/immich#2631] [Feature Request]
### The feature

After importing some photos, I found that the list of places in the explore view quickly becomes crowded. Therefore, I suggest grouping the places by countries to get a better overview. Clicking on a country should then open a list of tiles with all places that are in that country. An additional tile should offer the possibility to view all pictures of the selected country (new search "m:[country]").

Possible UI for mobile:
# <img alt="screenshot" src="https://i.imgur.com/1e95HwI.png" >

I also noticed that the web UI is currently limited to 12 items. It would be great to have a similar UI as on mobile, like a show all button which leads to listview.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by dominikheinz** [comment]
I agree, that would be a great feature to have! Especially when traveling through many countries, it would be nice to filter by each individual country.

**Comment by jrasm91** [comment]
I think it's a great idea. FWIW you can already do a country search via `m:country name`.  Also there is server support to filter by country, although it's not built into the app or website anywhere yet.

**Comment by kvalev** [comment]
I will take a stab at implementing that. I was thinking of designing it the same way as the albums page - have some controls at the top that allow for searching, sorting an grouping places. For places I guess the grouping strategy will be either None, meaning as it currently is or Country, where the places will be grouped similarly to how Albums are grouped by year.

**Comment by bo0tzz** [comment]
This is supported on web now.
